Add documentation for integration kaku_rc (#45388)
home-assistant/home-assistant.github.io: source/_integrations/klik_aan_klik_uit_rc.markdown

- Include KlikAanKlikUit devices in automations, scripts, scenes, and dashboards

## Supported devices

The following devices are known to be compatible:

**KlikAanKlikUit** self-learning devices, including but not limited to most plug-in and built-in KlikAanKlikUit outlets and dimmers.

**Intertechno** self-learning devices:
IT-1500, IT-2300, ITDL-1000, ITDM-250, ITL-1000, ITL-230, ITL-250, ITL-300, ITL-3500, ITL-500, ITLM-1000, ITLR-3500, ITR-1500, ITWR-3500, LBUR-100

**Nexa** self-learning plug-in outlets and dimmers.

**Telldus** self-learning plug-in outlets.

## Prerequisites

Before adding the integration, make sure:

1. You have a working RF transmitter in Home Assistant.
2. The transmitter supports 433.92 MHz OOK.
3. You know the device's values you want to use:
   - Device ID
   - Channel (unit number)
   - Whether you use a group command

## Configuration options

{% include integrations/config_flow.md %}

## Supported functionality

The **KlikAanKlikUit** integration provides the following entities.

### Switches

- **Power switch**
  - **Description**: Turns the device on or off.
  - **Available when**: Always.

## Data updates

> This integration does not poll device's state from the device.

Reviewer: ```suggestion
This integration does not {% term polling polls %} device's state from the device.
```
There's a [glossary term reference](https://developers.home-assistant.io/docs/documenting/general-style-guide#glossary-term-reference) for this term :book: .

Because KlikAanKlikUit RF control is one-way, Home Assistant cannot confirm whether a device actually changed state. The integration uses assumed state and restores the last known state after restart.

## Known limitations

- The protocol is one-way, so state changes made outside Home Assistant are not reported back.
- Learn and unlearn buttons are not created for group entries.
- Dimming support depends on your device's model.

## Pairing devices
- how to pair a device depends on it's design. Most of them have a learn-button. Push it > 2s until it's LED is flashing, then within 30s issue the integration's pairing function during configuration. Some devices go into learning-mode after repowering.

## Troubleshooting

### No compatible transmitter is shown

Make sure your RF transmitter integration is set up and supports 433.92 MHz OOK.

> ### Setup error for Device ID or Channel

Reviewer: You could use the Symptom/Resolution pattern here to give more context. See https://github.com/home-assistant/home-assistant.io/blob/current/source/_integrations/_integration_docs_template.markdown?plain=1#L203

---
title: KlikAanKlikUit
description: Instructions on how to integrate KlikAanKlikUit and compatible devices into Home Assistant.
ha_category:
  - Switch

> ha_release: 2026.6

Reviewer: ```suggestion
ha_release: 2026.7
```

ha_iot_class: Assumed State
ha_config_flow: true
ha_codeowners:
  - '@Phunkafizer'
ha_domain: klik_aan_klik_uit
ha_platforms:
  - switch
ha_integration_type: device
ha_quality_scale: bronze
---

The **KlikAanKlikUit** {% term integration %} lets you control self-learning KlikAanKlikUit 433.92 MHz RF devices from Home Assistant.

The integration uses the [Radio Frequency](/integrations/radio_frequency/) {% term entity %} platform to send RF commands. That means you first need a compatible RF transmitter in Home Assistant, such as an ESPHome device with 433.92 MHz OOK transmit support.

## Use cases

You can use this integration to:

- Turn KlikAanKlikUit outlets on and off from Home Assistant
- Add brightness control for compatible dimmable devices

> - Pair or unpair devices from Home Assistant by using built-in learn and unlearn buttons

Reviewer: ```suggestion
```
I think these are no longer accurate for this PR.

- Include KlikAanKlikUit devices in automations, scripts, scenes, and dashboards

## Supported devices

The following devices are known to be compatible:

**KlikAanKlikUit** self-learning devices, including but not limited to most plug-in and built-in KlikAanKlikUit outlets and dimmers.

**Intertechno** self-learning devices:
IT-1500, IT-2300, ITDL-1000, ITDM-250, ITL-1000, ITL-230, ITL-250, ITL-300, ITL-3500, ITL-500, ITLM-1000, ITLR-3500, ITR-1500, ITWR-3500, LBUR-100

**Nexa** self-learning plug-in outlets and dimmers.

**Telldus** self-learning plug-in outlets.

## Prerequisites

Before adding the integration, make sure:

1. You have a working RF transmitter in Home Assistant.
2. The transmitter supports 433.92 MHz OOK.
3. You know the device's values you want to use:
   - Device ID
   - Channel (unit number)
   - Whether you use a group command

> ## Configuration options

Reviewer: ```suggestion
```

{% include integrations/config_flow.md %}

## Supported functionality

The **KlikAanKlikUit** integration provides the following entities.

### Switches

> - **Power switch**

Reviewer: ```suggestion
- **Output**
```
I couldn't find a string for "Power switch". It seems to be called output. Or am I looking in the wrong place?

  - **Description**: Turns the device on or off.
  - **Available when**: Always.

## Data updates

This integration does not poll device's state from the device.

Because KlikAanKlikUit RF control is one-way, Home Assistant cannot confirm whether a device actually changed state. The integration uses assumed state and restores the last known state after restart.

## Known limitations

- The protocol is one-way, so state changes made outside Home Assistant are not reported back.

> - Learn and unlearn buttons are not created for group entries.

Reviewer: ```suggestion
```
no buttons added in this PR

- Dimming support depends on your device's model.

## Pairing devices

> - how to pair a device depends on it's design. Most of them have a learn-button. Push it > 2s until it's LED is flashing, then within 30s issue the integration's pairing function during configuration. Some devices go into learning-mode after repowering.

Reviewer: ```suggestion
How to pair a device depends on its design. Most of them have a learn button. Push and hold it for > 2s until its LED starts flashing. Then, within 30 seconds, issue the integration's pairing function during configuration. Some devices go into learning mode after repowering.
```

---
title: KlikAanKlikUit RC
description: Instructions on how to integrate KlikAanKlikUit RC and compatible devices into Home Assistant.
ha_category:
  - Button

>   - Light

Reviewer: ```suggestion
```
if light is removed from the core PR, it should be removed here, too

  - Switch

> ha_release: 2026.5

Reviewer: ```suggestion
ha_release: 2026.6
```

ha_iot_class: Assumed State
ha_config_flow: true
ha_codeowners:
  - '@Phunkafizer'
ha_domain: klik_aan_klik_uit_rc
ha_platforms:

>   - light

Reviewer: ```suggestion
```

**Intertechno** self-learning devices:
IT-1500, IT-2300, ITDL-1000, ITDM-250, ITL-1000, ITL-230, ITL-250, ITL-300, ITL-3500, ITL-500, ITLM-1000, ITLR-3500, ITR-1500, ITWR-3500, LBUR-100

**Nexa** self-learning plug-in outlets and dimmers.

**Telldus** self-learning plug-in outlets.

## Prerequisites

Before adding the integration, make sure:

1. You have a working RF transmitter in Home Assistant.
2. The transmitter supports 433.92 MHz OOK.
3. You know the device's values you want to use:
   - Device ID
   - Channel (unit number)
   - Whether you use a group command

## Configuration options

After setup, you can reconfigure the entry from {% my integrations title="**Settings** > **Devices & services**" %}.

The reconfigure flow lets you update:

- Radio frequency transmitter
- Device ID
- Channel
- Group
- Dimming

## Supported functionality

The **KlikAanKlikUit RC** integration provides the following entities.

### Switches

- **Power switch**
  - **Description**: Turns the device on or off.
  - **Available when**: Always.

### Lights

- **Brightness**
  - **Description**: Controls device's brightness.
  - **Available when**: You enabled **Dimming** during setup.

> 

Reviewer: ```suggestion
```
If lights are removed from core, they should be removed here, too

  - **Available when**: Group is disabled.
- **Unlearn**
  - **Description**: Sends an unlearning signal to unpair a compatible device.
  - **Available when**: Group is disabled.

## Data updates

This integration does not poll device's state from the device.

Because KlikAanKlikUit RF control is one-way, Home Assistant cannot confirm whether a device actually changed state. The integration uses assumed state and restores the last known state after restart.

## Known limitations

- The protocol is one-way, so state changes made outside Home Assistant are not reported back.
- Learn and unlearn buttons are not created for group entries.
- Dimming support depends on your device's model.

## Pairing devices
- how to pair a device depends on it's design. Most of them have a learn-button. Push it > 2s until it's LED is flashing, then within 30s issue the integration's learn function. Some devices go into learning-mode after repowering.

## Troubleshooting

### No compatible transmitter is shown

Make sure your RF transmitter integration is set up and supports 433.92 MHz OOK.

### Setup error for Device ID or Channel

> Make sure the values are in range:

Reviewer: where do I check this?

Author: the config flow will say it.
